altair/examples/top_k_items.py

```diff
@@ -0,0 +1,32 @@
+"""
+Top K Items
+-------------
```

Comment:
```suggestion
-----------
```
Make sure the number of dashes matches up with the title